optimum/exporters/onnx/convert.py

```diff
@@ -22,20 +22,17 @@
 import numpy as np
 from transformers.utils import is_tf_available, is_torch_available
 
-from ...utils import logging
+from ...utils import TORCH_MINIMUM_VERSION, is_torch_onnx_support_available, logging
+from ..tasks import TasksManager
 from .base import OnnxConfig
-from .utils import (
-    MIN_TORCH_VERSION,
-    get_decoder_models_for_export,
-    get_encoder_decoder_models_for_export,
-    is_torch_onnx_support_available,
-)
 
 
 if is_torch_available():
     from transformers.modeling_utils import PreTrainedModel
     from transformers.pytorch_utils import is_torch_less_than_1_11
 
+    from diffusers import ModelMixin
```

Comment:
We do not need a `is_diffusers_available`? 
I actually do not know since we add `diffusers` as extras for `exporters`...
Feel free to ignore!

Reply:
We don't since `diffusers` was added as an extra for `exporters`